src/containers/dashboard/purchases.svelte

```diff
@@ -0,0 +1,29 @@
+<script>
+  import Card from 'ui/card.svelte';
+  import Button from 'ui/button.svelte';
+  import PurchaseCard from '@/components/dashboard/purchase-card.svelte';
+
+  export let purchases = [];
+</script>
+
+<Card classname="card purchases">
+  <div class="title">
+    <svg src="/images/icons/shopping-bag.svg" class="icon" />
+    InnoStore purchases
+  </div>
+  {#if purchases.length}
+    <ul class="purchases-list">
+      {#each purchases as purchase (purchase.name + purchase.type)}
```

Comment:
```js
ongoing = (purchase) => purchase.status !== 'rejected' && purchase.status !== 'delivered';
```
```suggestion
      {#each purchases.filter(ongoing) as purchase (purchase.id)}
```

Reply:
Except of course if we add an option in the backend to send only ongoing purchases, which makes more sense (after I gave it 2 extra seconds of thought 😅)

Reply:
Excuse me, what on earth is an ongoing puchase

Reply:
forgive the stupid name in the example. I meant pending: something that isn't yet delivered nor is rejected

Reply:
Please, let's keep `ongoing` to projects, this is important) you guys seem confused enough